Makefile

```diff
@@ -183,6 +183,10 @@ run-ci-e2e-test:
 run-ci-e2e-upgrade-test:
 	hack/run-ci-e2e-test.sh -t upgrade
 
+.PHONY: run-ci-e2e-byoh-test $(ARGS)
+run-ci-e2e-byoh-test:
+	hack/run-ci-e2e-test.sh ${ARGS}
+
 .PHONY: clean
```

Comment:
```
[windows-machine-config-operator]$ make run-ci-e2e-byoh-test ARGS="-t basic -m 0 -c 1"
hack/run-ci-e2e-test.sh -t basic -m 0 -c 1
ARTIFACT_DIR is not set. Artifacts will be stored in: /tmp/tmp.FojjLu5RUr
make[1]: Entering directory '/home/jvaldes/Projects/windows-machine-config-operator'
go fmt ./...
go vet ./...
build/build.sh "build/_output" 5.0.0 -mod=vendor
building windows-machine-config-operator...
make[1]: Leaving directory '/home/jvaldes/Projects/windows-machine-config-operator'
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
INFO[0000] All validation tests have completed successfully 
Error from server (NotFound): namespaces "openshift-windows-machine-config-operator" not found
namespace/openshift-windows-machine-config-operator created
namespace/openshift-windows-machine-config-operator labeled
INFO[0001] Creating windows-machine-config-operator registry 
INFO[0001]   Creating ConfigMap "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-manifests-package" 
INFO[0001]   Creating ConfigMap "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-manifests-5-0-0" 
INFO[0001]   Creating Deployment "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" 
INFO[0001]   Creating Service "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" 
INFO[0001] Waiting for Deployment "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" rollout to complete 
INFO[0002]   Waiting for Deployment "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" to rollout: 0 out of 1 new replicas have been updated 
INFO[0013]   Waiting for Deployment "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" to rollout: 0 of 1 updated replicas are available 
INFO[0023]   Deployment "openshift-windows-machine-config-operator/windows-machine-config-operator-registry-server" successfully rolled out 
INFO[0023] Created CatalogSource: windows-machine-config-operator-catalog 
INFO[0023] OperatorGroup "operator-sdk-og" created      
INFO[0023] Created Subscription: windows-machine-config-operator-v5-0-0-sub 
INFO[0026] Approved InstallPlan install-w7fg8 for the Subscription: windows-machine-config-operator-v5-0-0-sub 
INFO[0026] Waiting for ClusterServiceVersion "openshift-windows-machine-config-operator/windows-machine-config-operator.v5.0.0" to reach 'Succeeded' phase 
INFO[0026]   Waiting for ClusterServiceVersion "openshift-windows-machine-config-operator/windows-machine-config-operator.v5.0.0" to appear 
INFO[0028]   Found ClusterServiceVersion "openshift-windows-machine-config-operator/windows-machine-config-operator.v5.0.0" phase: Pending 
INFO[0031]   Found ClusterServiceVersion "openshift-windows-machine-config-operator/windows-machine-config-operator.v5.0.0" phase: Installing 
INFO[0042]   Found ClusterServiceVersion "openshift-windows-machine-config-operator/windows-machine-config-operator.v5.0.0" phase: Succeeded 
INFO[0042] OLM has successfully installed "windows-machine-config-operator.v5.0.0" 
deployment "windows-machine-config-operator" successfully rolled out
configmap/windows-instances created
=== RUN   TestWMCO
=== RUN   TestWMCO/operator_deployed_without_private_key_secret
--- PASS: TestWMCO (1.32s)
    --- PASS: TestWMCO/operator_deployed_without_private_key_secret (0.42s)
PASS
ok      github.com/openshift/windows-machine-config-operator/test/e2e   1.554s
?       github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo       [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/aws     [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/azure   [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/none    [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/vsphere [no test files]
=== RUN   TestWMCO
=== RUN   TestWMCO/create
=== RUN   TestWMCO/create/Creation
=== RUN   TestWMCO/create/Creation/Windows_instance_configured_by_ConfigMap_controller
2022/01/03 20:19:43 waiting for 0/1 Windows nodes
2022/01/03 20:20:43 waiting for 0/1 Windows nodes
2022/01/03 20:21:43 waiting for 0/1 Windows nodes
2022/01/03 20:22:43 waiting for 0/1 Windows nodes
2022/01/03 20:23:43 waiting for 0/1 Windows nodes
2022/01/03 20:24:43 waiting for 0/1 Windows nodes
2022/01/03 20:25:43 waiting for 0/1 Windows nodes
2022/01/03 20:26:43 waiting for 0/1 Windows nodes
2022/01/03 20:27:43 waiting for 0/1 Windows nodes
2022/01/03 20:28:43 waiting for 0/1 Windows nodes
2022/01/03 20:29:43 waiting for 0/1 Windows nodes
2022/01/03 20:30:43 waiting for 0/1 Windows nodes
2022/01/03 20:31:43 waiting for 0/1 Windows nodes
2022/01/03 20:32:43 waiting for 0/1 Windows nodes
2022/01/03 20:33:43 waiting for 0/1 Windows nodes
2022/01/03 20:34:43 waiting for 0/1 Windows nodes
2022/01/03 20:35:43 waiting for 0/1 Windows nodes
2022/01/03 20:36:43 waiting for 0/1 Windows nodes
2022/01/03 20:37:43 waiting for 0/1 Windows nodes
2022/01/03 20:38:43 waiting for 0/1 Windows nodes
2022/01/03 20:39:43 waiting for 0/1 Windows nodes
2022/01/03 20:40:43 waiting for 0/1 Windows nodes
2022/01/03 20:41:43 waiting for 0/1 Windows nodes
2022/01/03 20:42:43 waiting for 0/1 Windows nodes
2022/01/03 20:43:43 waiting for 0/1 Windows nodes
2022/01/03 20:44:43 waiting for 0/1 Windows nodes
2022/01/03 20:45:43 waiting for 0/1 Windows nodes
2022/01/03 20:46:43 waiting for 0/1 Windows nodes
2022/01/03 20:47:43 waiting for 0/1 Windows nodes
2022/01/03 20:48:43 waiting for 0/1 Windows nodes
2022/01/03 20:48:43 waiting for 0/1 Windows nodes
2022/01/03 20:48:43 30m0.266048877s time is required to configure 1 nodes
    create_test.go:74: 
                Error Trace:    create_test.go:74
                Error:          Received unexpected error:
                                timed out waiting for the condition
                Test:           TestWMCO/create/Creation/Windows_instance_configured_by_ConfigMap_controller
                Messages:       Windows node creation failed
--- FAIL: TestWMCO (1817.31s)
    --- FAIL: TestWMCO/create (1816.45s)
        --- FAIL: TestWMCO/create/Creation (1816.45s)
            --- FAIL: TestWMCO/create/Creation/Windows_instance_configured_by_ConfigMap_controller (1800.27s)
FAIL
FAIL    github.com/openshift/windows-machine-config-operator/test/e2e   1817.534s
?       github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo       [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/aws     [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/azure   [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/none    [no test files]
?       github.com/openshift/windows-machine-config-operator/test/e2e/providers/vsphere [no test files]
FAIL
make: *** [Makefile:188: run-ci-e2e-byoh-test] Error 1

```

While validating with a development `platform=none` UPI cluster, the e2e test suit creates the `windows-instances` configMap and supports platform=none as cloud provider. No Windows instances are getting created via machineSet generation.

Reply:
WMCO was not able to provision the BYOH Windows instance as a Windows node due to CSR approval issues.

```
ERROR	controller-runtime.manager.controller.certificatesigningrequest	Reconciler error
  {
    "reconciler group": "certificates.k8s.io",
    "reconciler kind": "CertificateSigningRequest",
    "name": "csr-47vzn",
    "namespace": "",
    "error": "WMCO CSR Approver could not approve csr-47vzn CSR:
        could not validate contents for approval of CSR: csr-47vzn:
        error validating node name winhost for CSR: csr-47vzn:
        unable to map node name to the addresses of Windows instances:
        failed to lookup   DNS for IP 192.168.20.13:
        lookup 13.20.168.192.in-addr.arpa. on 192.168.20.2:53: no such host",
        ...
}
```

A PTR record must exist corresponding to the instance address (13.20.168.192 in this case) which resolves to the instance hostname for successful reverse DNS lookups.